be/src/exec/csv_scan_node.cpp

```diff
@@ -672,14 +672,14 @@ void CsvScanNode::hll_hash(const char* src, int len, std::string* result) {
     std::string str(src, len);
     if (str != "\\N") {
         uint64_t hash = HashUtil::murmur_hash64A(src, len, HashUtil::MURMUR_SEED);
-        char buf[HllHashFunctions::HLL_INIT_EXPLICT_SET_SIZE];
+        char buf[10];
```

Comment:
We should use a const or a macro, better not to use a magic number

Reply:
I see. Use magic number only for simple.
Because I think we could delete this class code in doris 0.12 version.